tests/core/full_node/stores/test_block_store.py

```diff
@@ -300,7 +300,7 @@ async def test_get_generator(bt: BlockTools, db_version: int, use_cache: bool) -
     blocks = bt.get_consecutive_blocks(10)
 
     def generator(i: int) -> SerializedProgram:
-        return SerializedProgram.from_bytes(int_to_bytes(i))
+        return SerializedProgram.from_bytes(int_to_bytes(i + 1))
```

Comment:
`int_to_bytes(0)` is an empty buffer, which is not a valid CLVM program. adding `1` avoids this issue.